crates/evm_mlir/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
remove

Reply:
Removed in [958f57a](https://github.com/lambdaclass/ethereum_rust/pull/548/commits/958f57aa2de82a68751afadfe4c52d3897385073)